core/database/foxx/api/support.js

```diff
@@ -1844,7 +1844,7 @@ module.exports = (function() {
                 }
 
                 // User ACL first
-                if (parent.acls && ((parent.acls & 1) != 0)) {
```

Comment:
@nedvedba do you understand what this is doing? I don't, it would be good to have an explanation of what is in parent.acls before approving this.